docs/team/longnguyentan.md

```diff
@@ -19,9 +19,10 @@ Given below are my contributions to the project.
 * **Contributions to the UG**:
   * Wrote Introduction & resized images ([#165](https://github.com/AY2324S1-CS2103T-T14-2/tp/pull/165)
   * Updated `list` and `delete` features ([#51](https://github.com/AY2324S1-CS2103T-T14-2/tp/pull/51)
+  * Updated features: Find, Help, List, Clear, Exit
```

Comment:
Good that you included your contribution. Would it be helpful if you add links to pr that shows it? Not really sure how our TA will assess

Reply:
I'll update it when it's close to the deadline